Emby.Server.Implementations/Emby.Server.Implementations.csproj

```diff
@@ -34,7 +34,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Pretty sure `netstandard2.1` will have the required apis, maybe we should wait?

Reply:
We should just use the rtf2898 derivation instead of a simple hash. (The one I linked above)

That is in netstandard2.0

Reply:
Netstandard2.0 does have that, and indeed I am using it. I could theoretically do some shenanigans to greatly reduce the supported sets of hashing algos and keep it on netstandard, but aside from one piece of bookkeeping when we major version rev, what is the major gain for this specific subproject. Its not used elsewhere like a library to the best of my knowledge, having it able to cross compile offers us little in exchange for a less frustrating and more performant (according to the doc) set of crypto libs.

Reply:
We need it to be `netstandard` if we want to create a `.app` for macos with Xamarin.

Reply:
If that's necessary than fine. I've downgraded it to netstandard 2.0. What this means is for the time being the hash method portion of all this is meaningless and it will always just do hmacsha1 as the hashing type for PBKDF2.